helm/ako/values.yaml

```diff
@@ -40,6 +40,11 @@ NetworkSettings:
   subnetPrefix: "" # Subnet Prefix of the vip network
   networkName: "" # Network Name of the vip network
   enableRHI: false # This is a cluster wide setting for BGP peering.
+  vipNetworkList: []
+  #vipNetworkList: []
+  #  - networkName: subnet-xxxxxxx
+  #  - subnetIP: 12.0.0.0
```

Comment:
This "-" is probably not required? Since you will have one map of elements within a slice?

Reply:
the user can either provide the networkName OR subnetIP, subnetPrefix pair. The vipNetworkList in the example has 2 elements, one of each.